common/persistence/sql/sqlplugin/tests/history_execution_request_cancel_test.go

```diff
@@ -0,0 +1,294 @@
+package tests
```

Comment:
Please add license header. I probably missed this in previous PRs also. Just run `make copyright` to see which files don't have it and `go run ./cmd/tools/copyright/licensegen.go` should automatically fix all of them.

Reply:
good catch, will do